timemachine/fe/system.py

```diff
@@ -149,3 +142,29 @@ def U_fn(x):
             return bond_U(x) + angle_U(x) + torsion_U(x) + jnp.sum(Us_vdw) + jnp.sum(Us_coulomb) + chiral_U
 
         return U_fn
+
+
+class HostGuestSystem:
+
+    # utility system container
+
+    def __init__(self, bond, angle, torsion, chiral_atom, chiral_bond, nonbonded_guest_pairs, nonbonded_host_guest):
+        self.bond = bond
+        self.angle = angle
+        self.torsion = torsion
+        self.chiral_atom = chiral_atom
+        self.chiral_bond = chiral_bond
+        self.nonbonded_guest_pairs = nonbonded_guest_pairs
+        self.nonbonded_host_guest = nonbonded_host_guest
+
+    def get_U_fns(self):
+
+        return [
+            self.bond,
+            self.angle,
+            self.torsion,
+            self.chiral_atom,
+            self.chiral_bond,
+            self.nonbonded_guest_pairs,
+            self.nonbonded_host_guest,
+        ]
```

Comment:
nit: could be a dataclass? (can use `Any` for attributes that don't have well-defined types)

Reply:
probably - but I think I'd rather you do this correctly (along with the `VacuumSystem` class).